openshift4/tusd/base/service.tusd.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    description: Exposes and load balances the application pods
+  creationTimestamp: null
+  labels:
+    app: tusd
+    app.kubernetes.io/component: documents
+    app.kubernetes.io/instance: tusd
+    app.kubernetes.io/managed-by: template
+    app.kubernetes.io/name: tusd
+    app.kubernetes.io/part-of: Core
+    cluster-name: tusd
+    phase: deploy
+  name: tusd-master
```

Comment:
I think just `tusd` for the name. `-master` comes from the patroni approach where one pod is considered a master/main